src/System.IO.FileSystem/src/System/IO/FileStream.Unix.cs

```diff
@@ -45,7 +45,7 @@ private SafeFileHandle OpenHandle(FileMode mode, FileShare share, FileOptions op
                 _asyncState = new AsyncState();
 
             // Translate the arguments into arguments for an open call.
-            Interop.Sys.OpenFlags openFlags = PreOpenConfigurationFromOptions(mode, _access, options); // FileShare currently ignored
+            Interop.Sys.OpenFlags openFlags = PreOpenConfigurationFromOptions(mode, _access, share, options); // FileShare currently ignored
```

Comment:
@tmds, can you remove the changes to these files from this PR?  It'll conflict with Jan's change to delete the files entirely.